app/controllers/projects_controller.rb

```diff
@@ -21,6 +21,7 @@ def new
   end
 
   def show
+    @links = policy_scope Link
```

Comment:
biztosan az összes linket szeretnénk mutatni, amit a user láthat? jobb lenne csak az aktuális project linkjeit scope-olni